src/__tests__/__snapshots__/index.to-string-class-name.js.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`allows you to use a component as a child selector 1`] = `
+.css-1o8fywl .g-1-239,
+[data-css-1o8fywl] .g-1-239 {
```

Comment:
I wonder if there's an easy way to reset this between tests 🤔  Otherwise, this snapshot's going to break anytime you add/remove glamorous components anywhere in the codebase 😢